documentation/docs/reference/nomad.md

```diff
@@ -0,0 +1,101 @@
+# Nomad configuration (Percona teams)
+
+Percona Monitoring and Management (PMM) includes HashiCorp Nomad to enable future extensibility and enhanced service capabilities.
+
+[Nomad](https://www.nomadproject.io/) is a workload orchestrator designed to deploy and manage containers and non-containerized applications. In PMM, Nomad provides the underlying infrastructure to:
+
+- improve resource allocation across monitoring components
+- enable future PMM extensibility 
+- manage distributed monitoring agents more efficiently
+
+Nomad is **disabled by default** in PMM and has no impact on system performance when not enabled. 
+
+### Internal configuration
+
+If you're a Percona team member or an advanced user who needs to enable Nomad for specific use cases, follow these steps:
+
+### Enable Nomad
+
+To enable Nomad in PMM:
+{ .power-number }
+
+1. Start PMM Server with the `PMM_ENABLE_NOMAD` environment variable:
+   ```
+   -e PMM_ENABLE_NOMAD=1
+   ```
+
+2. Expose the Nomad port:
+   ```
+   -p 4647:4647
+   ```
+
+3. Go to PMM's **Advanced Settings** and set the public address.
+
+??? info "Docker run command" 
+
+    ```
+    docker run -d \
+    -e PMM_ENABLE_NOMAD=1 \
+    -p 4647:4647 \
+    -p 443:443 \
+    --name pmm-server \
+    percona/pmm-server:3
+    ```
+
+### Disable Nomad
+
+To disable Nomad:
+
+```
+-e PMM_ENABLE_NOMAD=0
+```
+
```

Comment:
I think we can mention that removing Public Adress on Settings page also disables Nomad @BupycHuk what do you think?

Reply:
I would mention on previous section that public address set is a pre-requisite for nomad to work